src/twisted/test/test_nooldstyle.py

```diff
@@ -145,7 +150,13 @@ def test_newStyleClassesOnly(self):
                     oldStyleClasses.append(fullyQualifiedName(val))
 
         if oldStyleClasses:
+
             self.todo = "Not all classes are made new-style yet. See #8243."
+
+            for x in forbiddenModules:
```

Comment:
Just a note -- this is red because there are no forbidden modules that have old-style classes in them (this should always be red)